ledger/src/blockstore_processor.rs

```diff
@@ -635,15 +631,6 @@ pub fn process_blockstore_from_root(
     cache_block_meta_sender: Option<&CacheBlockMetaSender>,
     accounts_background_request_sender: &AbsRequestSender,
 ) -> result::Result<(), BlockstoreProcessorError> {
-    if let Some(num_threads) = opts.override_num_threads {
```

Comment:
Thanks for also taking care of an unused argument after we've moved to a shared static thread pool.